<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Ajay3147361" name="lalaTestNGTest" tests="23" failures="2" timestamp="5 Dec 2018 07:58:16 GMT" time="5.059" errors="0">
  <testcase name="verifyFutureOrder" time="1.434" classname="com.lala.test.PlaceOrderTest">
    <failure type="java.lang.AssertionError" message="Test is failed, Error while placing order, Please check by placing order manually.">
      <![CDATA[java.lang.AssertionError: Test is failed, Error while placing order, Please check by placing order manually.
	at com.lala.test.PlaceOrderTest.verifyFutureOrder(PlaceOrderTest.java:83)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- verifyFutureOrder -->
  <testcase name="verifyNewOrder" time="1.114" classname="com.lala.test.PlaceOrderTest"/>
  <testcase name="verifyNewOrderWithInvalidPayload" time="0.025" classname="com.lala.test.PlaceOrderTest"/>
  <testcase name="verifyFetchOrderForNewOrder" time="0.992" classname="com.lala.test.FetchOrderTest"/>
  <testcase name="verifyFetchOrderWhenOrderDoesNotExist" time="0.031" classname="com.lala.test.FetchOrderTest"/>
  <testcase name="verifyFetchOrderWhenOrderExist" time="0.02" classname="com.lala.test.FetchOrderTest"/>
  <testcase name="verifyDriveToTakeAlreadyCancelledOrder" time="0.041" classname="com.lala.test.DriveToTakeOrderTest"/>
  <testcase name="verifyDriveToTakeAlreadyCompletedOrder" time="0.044" classname="com.lala.test.DriveToTakeOrderTest"/>
  <testcase name="verifyDriveToTakeAlreadyOnGoingOrder" time="0.035" classname="com.lala.test.DriveToTakeOrderTest"/>
  <testcase name="verifyDriveToTakeExistingOrder" time="0.019" classname="com.lala.test.DriveToTakeOrderTest">
    <failure type="java.lang.AssertionError" message="expected [200] but found [422]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [422]
	at com.lala.test.DriveToTakeOrderTest.verifyDriveToTakeExistingOrder(DriveToTakeOrderTest.java:85)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- verifyDriveToTakeExistingOrder -->
  <testcase name="verifyDriveToTakeOnOrderDoesNotExist" time="0.014" classname="com.lala.test.DriveToTakeOrderTest"/>
  <testcase name="verifyDriveToTakeOrder" time="0.374" classname="com.lala.test.DriveToTakeOrderTest"/>
  <testcase name="verifyDriveToCompleteForCancelOrder" time="0.036" classname="com.lala.test.DriveToCompleteOrderTest"/>
  <testcase name="verifyDriveToCompleteForCompletedOrder" time="0.045" classname="com.lala.test.DriveToCompleteOrderTest"/>
  <testcase name="verifyDriveToCompleteForNewOrder" time="0.305" classname="com.lala.test.DriveToCompleteOrderTest"/>
  <testcase name="verifyDriveToCompleteForNonExistingOrder" time="0.015" classname="com.lala.test.DriveToCompleteOrderTest"/>
  <testcase name="verifyDriveToCompleteForOnderOnAssignedState" time="0.041" classname="com.lala.test.DriveToCompleteOrderTest"/>
  <testcase name="verifyCancelExistingOrder" time="0.021" classname="com.lala.test.CancelOrderTest"/>
  <testcase name="verifyCancelNewOrder" time="0.29" classname="com.lala.test.CancelOrderTest"/>
  <testcase name="verifyCancelNonExistingOrder" time="0.015" classname="com.lala.test.CancelOrderTest"/>
  <testcase name="verifyCancelOrderForAlreadyCancelledOrder" time="0.028" classname="com.lala.test.CancelOrderTest"/>
  <testcase name="verifyCancelOrderForCompletedOrder" time="0.028" classname="com.lala.test.CancelOrderTest"/>
  <testcase name="verifyCancelOrderForOnGoingOrder" time="0.02" classname="com.lala.test.CancelOrderTest"/>
</testsuite> <!-- lalaTestNGTest -->
